sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/InterfacesShouldNotBeEmpty.cs

```diff
@@ -45,17 +45,24 @@ protected override void Initialize(SonarAnalysisContext context)
                 c =>
                 {
                     var interfaceDeclaration = (InterfaceDeclarationSyntax)c.Node;
-                    var interfaceSymbol = c.SemanticModel.GetDeclaredSymbol(interfaceDeclaration);
+                    if (interfaceDeclaration.Identifier.IsMissing ||
+                        interfaceDeclaration.Members.Count > 0)
```

Comment:
The analyser should be a little faster if we don't immediately go for the symbol